### PR TITLE
Run generate-babel-types-docs on ci

### PR DESCRIPTION
**Pull request by sarupbanskota**
Started a fresh branch from `7.0`, ran the script, and found some changes => must mean that it wasn't run the last time an update to the script was made on `7.0`

Instead of reporting an issue, just thought to make a PR, feel free to reject if it's already done and I missed something.

### CODE REVIEW

**Comment by hzoo** [comment]
If we run it on ci we'd need to commit it too? Or can we just run it as part of pre-commit?

**Comment by sarupbanskota** [comment]
:man_facepalming: what was I thinking? :) 

I'll send an update tomorrow!

**Comment by jridgewell** [comment]
This can be a check that fails if there are updates.

**Comment by sarupbanskota** [comment]
Nice idea @jridgewell 

Here we go, I've now made it fail CI if docs aren't updated. Note how the first commit fails CI, and the second one passes it.

I couldn't find an easy way to throw an error, but I think thanks to script's filename, the message explains itself:

``` bash
# from the CI error

make check-babel-types-docs
sh ./scripts/check-babel-types-docs.sh
make[1]: Entering directory `/home/ubuntu/babel'
node ./scripts/generate-babel-types-docs.js
make[1]: Leaving directory `/home/ubuntu/babel'
packages/babel-types/README.md babel-types_README differ: byte 2271, line 109
make: *** [check-babel-types-docs] Error 1

make check-babel-types-docs returned exit code 2
```

Please lemme know if you'd like me to move to a different CI etc

File: Makefile [hunk 1]
```diff
@@ -58,6 +58,9 @@ test-ci:
 	make bootstrap
 	make test-only
 
+update-babel-types-docs:
```
Comment (jridgewell) [inline Makefile]: rename to `check-babel-types-docs`

File: scripts/check-babel-types-docs.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+
+cp packages/babel-types/README.md babel-types_README
+make babel-types-docs
+
+cmp --silent packages/babel-types/README.md babel-types_README
```
Comment (jridgewell) [inline scripts/check-babel-types-docs.sh]: let's remove the copy afterwards

File: Makefile (hunk 1 above)
Comment (sarupbanskota) [inline Makefile]: I actually called it `check-babel-types-docs` before, but renamed it as the error sounded better when it complained about `update...` .. but I'll change it back.

File: scripts/check-babel-types-docs.sh (hunk 2 above)
Comment (sarupbanskota) [inline scripts/check-babel-types-docs.sh]: Done

File: Makefile (hunk 1 above)
Comment (jridgewell) [inline Makefile]: "assert"?

File: scripts/check-babel-types-docs.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+rm babel-types_README
```
Comment (jridgewell) [inline scripts/check-babel-types-docs.sh]: Isn't this the return value now?

**Comment by nicolo-ribaudo** [comment]
`@babel/types` docs are not in this repo anymore. Thanks for your PR anyway!

**Pull request closed without merging**